README.md

```diff
@@ -40,6 +40,29 @@ Then call Ansible:
 
     ANSIBLE_LOAD_CALLBACK_PLUGINS=1 ansible localhost -m ping -vvv
 
+### Puppet
+
+To install and configure a Puppetserver on EL7, run the following:
+
+```bash
+# Install the server - modify as needed for your platform
+yum -y install https://yum.puppet.com/puppet7-release-el-7.noarch.rpm
```

Comment:
Worked for EL8 as well.

Reply:
Yes, that's what I meant with modify as needed.